determined-ai/determined
# feat: replace profiler empty page

## Description

### Before

<img width="1728" alt="image" src="https://github.com/determined-ai/determined/assets/109113616/5d0ba4e7-3bfb-4a2c-8a37-d1214a3b3b3b">


### After

<img width="1728" alt="image" src="https://github.com/determined-ai/determined/assets/109113616/87dc8f77-f2e1-4c7e-ae2e-8eb14dbb8bc0">






## Test Plan
- Check if all texts are followed by [WEB-954]
- Check if the link jumps to [the right page](http://latest-main.determined.ai:8080/docs/reference/reference-training/experiment-config-reference.html#profiling)




## Commentary (optional)





## Checklist

  See [Release Note](https://github.com/determined-ai/determined/blob/master/docs/release-notes/README.md) for details.

## Ticket
